aljuarismi/operations.py

```diff
@@ -36,8 +36,12 @@ def do_clustering(parameters, dataset):
     :return:
     """
     op = parameters.pop("operation")
-    if parameters["Dataset"]:
-        dataset = al.Workspace().get_dataset(parameters["Dataset"])
+    workspace = al.Workspace()
+    data_name = parameters["Dataset"]
+    if data_name:
+        dataset = workspace.get_dataset(data_name)
+    if dataset is None:
```

Comment:
I would do:
```
if not None:
    dataset = workspace.get_dataset(data_name)
```

Reply:
The existence of the dataset is check out in main

Reply:
This check if the dataset that we want to use exists or not